rfcs/convergence/authoring-stories.md

```diff
@@ -267,13 +270,40 @@ TBA
 
 TBA
 
-### 7. how structure stories in the storybook nav tree
+### 7. dissecting big story files into smaller ones
 
-TBA
+Components with complex API require many long stories, both for documentation and for e2e testing. This creates long files with hundres of lines, which are hard to maintain. How can these files be dissected into smaller ones?
+
+**Proposal:**
 
-### 8. dissecting big story files into smaller ones
+1. Every component can have at most one `.stories.tsx` file. This file has a default export with Storybook configuration for the docs page.
+2. If putting all stories into one file would make it too long, individual stories might be put into `.story.tsx` files as a named export, and then re-exported from `.stories.tsx` file like this: `export * from ‘./IndividualStoryFile.story’;`
```

Comment:
Again, how can we make sure people use the `story.tsx` convention ? what stops me from using a `.tsx` file ... which would be terrible and quite confusing

Reply:
Unfortunately, I don’t see a way enforce this. I still think it’s good to have documented what’s the right way of dealing with large files, so we could spread the knowledge and refer to it during PR review.

Reply:
also note that only trigger point that tells storybook to render anything is the presence if `CSF` default export - which makes need of 2 different suffixes not relevant

Reply:
> I'd suggest to stay away from having 2 file name suffixes for storybook runtime code (.stories.* and *.story) -> *.stories.* should be sufficient

:white_check_mark: done